### PR TITLE
Docs(web,web-twig,web-react): Migration Guides cleanup #DS-1250

### PR DESCRIPTION
**Pull request by adamkudrna**

### CODE REVIEW

File: docs/migrations/web-react/MIGRATION-v2.md [hunk 1]
```diff
@@ -174,224 +144,330 @@ const onToggle = () => setIsOpen(!isOpen);
...
+  ```
+  `columnStart` = 1 + (12 - 6) / 2 = 4
+- Responsive grid item centered over 8 columns on mobile, 6 columns on tablet, and 4 columns on desktop:
+`diff
+    -<GridSpan over="8" tablet="6" desktop="4">…</GridSpan>
+    +<GridItem
+    +  columnStart="{{ mobile: 3, tablet: 4, desktop: 5 }}"
+    +  columnEnd="{{ mobile: 'span 8', tablet: 'span 6', desktop: 'span 4' }}"
+    +>…</GridItem>
+    ` - `columnStart` = 1 + (12 - 8) / 2 = 3 - `columnStart` = 1 + (12 - 6) / 2 = 4 - `columnStart` = 1 + (12 - 4) / 2 = 5
```
Comment (crishpeen) [inline docs/migrations/web-react/MIGRATION-v2.md:288]: this is somehow broken

File: docs/migrations/web-react/MIGRATION-v2.md [hunk 2]
```diff
@@ -174,224 +144,330 @@ const onToggle = () => setIsOpen(!isOpen);
...
+  ```
+  `columnStart` = 1 + (12 - 4) / 2 = 5
+- Centered grid item over 6 columns:
+  ```diff
+  -<GridSpan over="6">…</GridSpan>
+  +<GridItem columnStart="4" columnEnd="span 6">…</GridItem>
+  ```
+  `columnStart` = 1 + (12 - 6) / 2 = 4
+- Responsive grid item centered over 8 columns on mobile, 6 columns on tablet, and 4 columns on desktop:
+`diff
```
Comment (crishpeen) [inline docs/migrations/web-react/MIGRATION-v2.md:282]: probably not enough backticks

File: docs/migrations/web-twig/MIGRATION-v3.md [hunk 3]
```diff
@@ -147,34 +154,50 @@ Replace `tablet` and `desktop` props with the `cols` object prop.
...
+  ```
+  `columnStart` = 1 + (12 - 4) / 2 = 5
+- Centered grid item over 6 columns:
+  ```diff
+  -<GridSpan over="6">…</GridSpan>
+  +<GridItem columnStart="4" columnEnd="span 6">…</GridItem>
+  ```
+  `columnStart` = 1 + (12 - 6) / 2 = 4
+- Responsive grid item centered over 8 columns on mobile, 6 columns on tablet, and 4 columns on desktop:
+  ```diff
```
Comment (crishpeen) [inline docs/migrations/web-twig/MIGRATION-v3.md:187]: it looks correct here

**Review by crishpeen — APPROVED** [review]

**Review by literat — APPROVED** [review]
Thanks for the cleanup. Looks great 👍

**Review by curdaj — APPROVED** [review]

**Pull request merged**